polus-feature-extraction-plugin/src/main.py

```diff
@@ -1270,7 +1273,8 @@ def main():
             label_image,img_emb_unit = read(img_file[0][0]['file'])
             df = None
             files=''
-
+            channel=''
+            #Both intensity and labeled image passed as input
             if intDir:
                 #Get matching files
                 files = configfiles_int.get_matching(**{k.upper():v for k,v in img_file[0][0].items() if k not in ['file','c']})
```

Comment:
This is where you would do what I mentioned above. If `files` is empty, then throw a warning and move on to the next image.